homework/js-exercises/AboutMe/aboutMe.js

```diff
@@ -0,0 +1,16 @@
+document.body.setAttribute('style', 'font-family: Arial, sans-serif');
+document.getElementById('nickname').textContent = 'Lio';
+document.getElementById('fav-food').textContent = 'Falafel';
+document.getElementById('hometown').textContent = 'Heraklion';
+
+let listItems = Array.from(document.querySelectorAll('ul li'));
+
+
+for (let i in listItems) {
+    listItems[i].classList.add('list-item');
+}
```

Comment:
Since you have an array (`listItems`), it would better to use for..of or even better a .forEach

Reply:
Is it more "expensive" the way I do it or it is just more Javascript to do it the way you suggest?

Reply:
It is both. You are iterating an array like an object and then you access the array using the `listItems[i]` to get the item. Using for..of of .forEach you get the item directly.